src/main/java/org/spongepowered/api/item/inventory/Inventory.java

```diff
@@ -666,6 +666,17 @@
      */
     boolean containsInventory(Inventory inventory);
 
+    /**
+     * Transforms this inventory using the transformation
+     *
+     * @param transformation The Transformation
```

Comment:
Either use a lowercase `transformation` (recommended) or use a an `@link`. (same for the other references as well)